src/parser/core/CombatLogParser.js

```diff
@@ -562,13 +586,16 @@ class CombatLogParser {
     this.getModule(EventEmitter).addEventListener(...args);
   }
 
-  deepDisable(module) {
+  deepDisable(module, state = MODULE_ERROR.RESULTS) {
```

Comment:
Any particular reason why results is the default value? Could this be a required argument instead?

Reply:
oh yeah this is a remnant from when I didn't catch event handling yet so the deepDisable method was only used for results errors, I'll move it to a required argument